roles/openshift_logging/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
-openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default(
+  'registry.access.redhat.com/openshift3/ose-' if openshift_deployment_type == 'openshift-enterprise'
+  else 'docker.io/openshift/origin-') }}"
```

Comment:
what about the idea of using two variable files `enterprise` and `origin` and specify the default prefixes/versions in there?